src/content/docs/components/collapsible/readme.mdx

```diff
@@ -4,14 +4,23 @@ The `<Collapsible/>` component allows you to collapse/expand an element with arb
 and apply a nice transition to it. This is a challenge to achieve with CSS only, since it's impossible
 to transition the `height` property from `0` to `auto` and vice versa.
 
-We overcome this challenge by applying a height value based on the `clientHeight` of the content
-and then setting it to `auto` as soon as the transition is completed so that further changes
-in height are automatically applied. This essentially creates a transition between `0` and `auto`.
+This challenge is overcome by applying a *height* value based on the `clientHeight` of the content
+and then set it to `auto` as soon as the transition is complete so that further changes
```

Comment:
Should be `setting` to match the continuous tense of the verb `applying` mentioned previously.
Also, I think `completed` conveys the message a bit better, since it is a verb representing a process that has *finished*, rather than `complete`, which is a noun that represents something that is *whole*.

Reply:
I will make the adjustments and re-push